web-ui/src/main/resources/catalog/components/viewer/layermanager/partials/layerdimensions.html

```diff
@@ -152,3 +152,7 @@
     </datalist>
   </div>
 </div>
+
+<div class="form-group">
+  <label><input type="checkbox" data-ng-model="unsetTimeDimension" /><span data-translate="">unsetTimeDimension</span></label>
```

Comment:
If you add class `gn-margin-right-sm` to the `<input>` there will be a small space between checkbox and label